clusterloader2/testing/access-tokens/config.yaml

```diff
@@ -0,0 +1,137 @@
+# Stress testing access token validation
+#
+# Targeting 2 000 tokens with 5 000 total QPS for 5k node cluster, so it's 2.5 QPS per token.
+# For smaller cluster, we want to scale down lineary QPS per token to 2.5 * (Number of nodes)/(5 000).
```

Comment:
Could you make it more explicit that this test always creates 2K tokens and 80 pods and only the QPS depends on the cluster size?

Reply:
Added comment and calculations.